packages/date-picker/src/DatePicker/components/DateInput/DateInput.tsx

```diff
@@ -24,6 +24,11 @@ type OmittedInputProps =
   | "reversed"
   | "onBlur"
   | "value"
+  | "type"
+  | "ariaLabel"
+  | "ariaDescribedBy"
+  | "defaultInputValue"
+  | "automationId"
```

Comment:
```suggestion
  | "automationId"
  | "status"
```

Looks like `InputProps` also has a status of its own and is used to style the input itself. TypeScript didn't pick up on the conflict because the values just happen to be the same as the validation ones, but it's safer to omit it here so if there's any change in the future to either we don't get an error.